docs/adr/0002-container-cve-remediation.md

```diff
@@ -0,0 +1,34 @@
+# 2. Container CVE remediation
+
+Date: 2025-02-13
+
+## Status
+
+Accepted
+
+## Context
+
+There is a stakeholder request for MintMaker to inform the users if any container updates fix known vulnerabilities. These pull requests should be appropriately marked and should be created at a higher frequency - ignoring general container updates schedules. This functionality enables the users to quickly integrate new vulnerability fixes and expedite the release process.
+
+## Decision
+
+Upstream renovate already has a similar feature, osvVulnerabilityAlerts, which gathers the vulnerabilities from OSV. Since OSV doesn't publish vulnerability data for containers, a custom solution had to be implemented. 
```

Comment:
```suggestion
Upstream renovate already has a similar feature, `osvVulnerabilityAlerts`, which gathers the vulnerabilities from OSV. Since OSV doesn't publish vulnerability data for containers, a custom solution had to be implemented. 
```
Maybe add these as code to stand out?